lib/tasks/db.rake

```diff
@@ -0,0 +1,21 @@
+namespace :db do
+  desc 'Erase all tables'
+  task clear: :environment do
+    conn = ActiveRecord::Base.connection
+    tables = conn.tables
```

Comment:
@lostie this is a quick fudge for testing the importer and clearing and restoring the database while not in production - so comfortable with this for now.